pkg/crc/machine/machine.go

```diff
@@ -93,15 +93,19 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 		result.Status = vmState.String()
 	} else {
-		logging.InfoF(" Starting stopped VM ...")
 		host, err := libMachineAPIClient.Load(machineConfig.Name)
 		s, err := host.Driver.GetState()
 		if err != nil {
 			logging.ErrorF("Error getting the state for host: %s", err)
 			result.Error = err.Error()
 		}
+		if s == state.Running {
+			result.Status = s.String()
+			return *result, err
```

Comment:
what valuse has `err` here?

This is to shortcircuit the `start` when the VM is already in a running state

Reply:
Any error that might have occurred during getting the state of the VM :P 
Yup does not make sense.. if we already get into the if block.. will just return `nil` instead!

Reply:
In that case line 102 should be a return of the error... right?
as originally it just continues and would check the running state and try again, but if the previous call failed, that would fail also.

same actually here. it misses the return of the error

Reply:
@gbraad Updated return value to `nil`.